backend/src/main/java/pl/krkteam/authentication/RegistrationController.java

```diff
@@ -0,0 +1,46 @@
+package pl.krkteam.authentication;
+
+import com.google.gson.Gson;
+import pl.krkteam.player.Player;
+import pl.krkteam.player.PlayerConversionUtil;
+import pl.krkteam.player.PlayerDTO;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin(origins = "http://localhost:8080")
+@RestController
+public class RegistrationController {
+
+    PlayerHolder playerHolder = new PlayerHolder();
+
+    @PostMapping(value = "/login")
+    public String signPlayer(@RequestBody String post) {
+        System.out.println(post);
+
+        Gson gson = new Gson();
+
+        PlayerDTO playerDTO = gson.fromJson(post, PlayerDTO.class);
+
+        System.out.println(playerDTO);
```

Comment:
should not be in the production code

Reply:
Logger will be use at the end of this iteration